gtda/images/preprocessing.py

```diff
@@ -185,15 +185,31 @@ def plot(Xt, sample=0, colorscale='greys', origin='upper',
 
 @adapt_fit_transform_docs
 class Inverter(BaseEstimator, TransformerMixin, PlotterMixin):
-    """Invert all 2D/3D binary images in a collection.
+    """Invert all 2D/3D images in a collection.
+
+    Applies an inversion function to the value of all pixels of all images in
+    the input collection. The inversion function is defined as
+    :math:`f(x) = M - x`, where `x` is a pixel value and `M` is :attr:`max_value,
```

Comment:
Backtick missing after max_value.

More importantly (and sorry for not spotting this earlier) we should still say what inversion means in the boolean case (even if it is more trivial there)